manifests/mailserver.pp

```diff
@@ -275,6 +275,14 @@
       require => Package[$postfix_mysql_package]
     }
 
+    file { '/etc/postfix/update_vmail_database_for_security.sql':
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0444',
+      source  => 'puppet:///modules/atomia/mailserver/update_vmail_database_for_security.sql',
+      require => Package[$postfix_mysql_package]
+    }
+
```

Comment:
This part is not needed if you alter the `files/mailserver/mysql.schema.sql` file to include the salt column.

It may be better to create an upgrade shell script and reference it, so the script becomes available on all mail nodes for fast execution.